L1Trigger/L1TMuonEndCap/src/PtAssignmentEngine2017.cc

```diff
@@ -23,9 +23,11 @@ float PtAssignmentEngine2017::scale_pt(const float pt, const int mode) const {
   // TRG       = (1.2 + 0.015*TRG) * XML
   // TRG       = 1.2*XML / (1 - 0.015*XML)
   // TRG / XML = 1.2 / (1 - 0.015*XML)
-
-  if (ptLUTVersion_ >= 8) {  // First "physics" LUTs for 2022, will be deployed in June 2022
-    pt_xml = fmin(20., pt);  // Maximum scale set by muons with XML pT = 20 GeV (scaled pT ~32 GeV)
+  if (ptLUTVersion_ >= 9) {  // LUTs with lower pT scale for 2023, deployed in May 2023
+    pt_xml = fmin(20., pt);  // Maximum scale set by muons with XML pT = 20 GeV (scaled pT ~31 GeV)
+    pt_scale = 1.07 / (1 - 0.015 * pt_xml);
+  } else if (ptLUTVersion_ == 8) {  // First "physics" LUTs for 2022, will be deployed in June 2022
+    pt_xml = fmin(20., pt);         // Maximum scale set by muons with XML pT = 20 GeV (scaled pT ~32 GeV)
     pt_scale = 1.13 / (1 - 0.015 * pt_xml);
```

Comment:
Where are the 1.07 and 1.13 coming from? The block is otherwise very similar except for this constant. Is this something one might want to load or look up instead of writing block after block of LUT version `if`s?

Reply:
This is variable is not available in online payloads. This is the pT scale we apply to the LUT that's loaded at P5. We can technically o2o this fully, but that will require some rework. Also, it wouldn't help if we change the actual formula in the future (which is in our plans).

Instead we o2o the `pTLUTVersion_` such that we can specify which scaling to use in each case.

Reply:
Okay, then fine by me, but maybe for the next version we should consider whether this should be looked up or loaded.

Reply:
Another acceptable option would be to assign them to some named constant

Reply:
Yes, in principle this can be done. It wasn't planned to keep the same functional form throughout the years, that's why it wasn't implemented up to now. We are thinking of changing the form for the next iteration anyway, so I thought it would be ok to keep it like this. If you prefer that I move this to a named constant I can do that